src/forklift/core.py

```diff
@@ -87,9 +87,9 @@ def update(crate, validate_crate):
             has_custom = validate_crate(crate)
             if has_custom == NotImplemented:
                 check_schema(crate)
-        except ValidationException as e:
```

Comment:
can we get rid of the [validationexception](https://github.com/agrc/forklift/blob/2d3d207b9009f618579a9b871062890d8b901991/src/forklift/exceptions.py#L10:7) class?

Reply:
I had the same thought. But we do use it in core still.